test/k8s/simple_test.go

```diff
@@ -70,7 +70,28 @@ func Simple() {
 	})
 
 	AfterEach(func() {
-		time.Sleep(stabilizationSleep)
+		endpoint := cluster.GetPortForward("prometheus-server").ApiServerEndpoint
+		promClient, err := NewPromClient(fmt.Sprintf("http://%s", endpoint))
+		Expect(err).ToNot(HaveOccurred())
+
+		stopCh := make(chan struct{})
+		metricCh := make(chan int)
+		errCh := make(chan error)
+
+		go WatchXdsDeliveryCount(promClient, stopCh, metricCh, errCh)
+		defer close(stopCh)
+
+	Loop:
```

Comment:
I don't think I've ever seen a capitalized label before :D

Reply:
It seems like there is no strict guideline on this, internally golang is using both capitalized labels and not. See [examples](https://stackoverflow.com/questions/64039172/whats-the-standard-practice-for-naming-labels-in-go) in the first reply.

Reply:
yeah doesn't matter either way